src/main/java/study/ResultView.java

```diff
@@ -0,0 +1,13 @@
+package study;
+
+public class ResultView {
+
+	public void initStart() {
+		System.out.println("계산식을 입력하세요.");
+	}
+
+	public void viewResult(SimpleCalculator simpleCalculator) {
+		int result = simpleCalculator.calEquation();
+		System.out.println("계산 결과 = " + result);
+	}
```

Comment:
계산기 객체를 viewResult()의 인자로 넘기는 것 보다는 결과 값을 넘겨주는 것은 어떨까요?

계산기 객체와 View 객체를 분리하는 이유는 서로의 의존성을 끊어내고 변경의 영향을 최소화하기 위함입니다.

만약 계산기 객체의 calEquation() 메서드의 네이밍이 바뀌게 된다면 ResultView 에도 변경이 생기지 않을까요?

```suggestion
	public void viewResult(int result) {
		System.out.println("계산 결과 = " + result);
	}
```